1.process_data/nbconverted/process_and_normalize_jump_data.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Preprocess using pyctyominer and merge broad samples
+
+# ## Imports
+
+# In[ ]:
+
+
+import time
+from pathlib import Path
+import numpy as np
+import pandas as pd
+from pycytominer import normalize
+from pycytominer.cyto_utils.cells import SingleCells
+
+
+# ## Find the root of the git directory
+# This allows file paths to be referenced in a system agnostic way
+
+# In[ ]:
+
+
+# Get the current working directory
+cwd = Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
+
+# Check if a Git root directory was found
+if root_dir is None:
+    raise FileNotFoundError("No Git root directory found.")
+
+
+# ## Define Paths
+
+# In[ ]:
+
+
+# Input paths
+big_drive_path = f"{root_dir}/big_drive"
+sqlite_data_path = f"{big_drive_path}/data"
+ref_path = f"{root_dir}/1.process_data/reference_plate_data"
+barcode_platemap = f"{ref_path}/barcode_platemap.csv"
+
+# Output paths
+output_cell_count_path = Path(f"{big_drive_path}/sc_counts")
+normalized_path = Path(f"{big_drive_path}/normalized_sc_data")
+
+
+# ## Create directories if non-existent
+
+# In[ ]:
+
+
+output_cell_count_path.mkdir(parents=True, exist_ok=True)
+normalized_path.mkdir(parents=True, exist_ok=True)
+
+
+# In[ ]:
```

Comment:
It looks like this notebook isn't executed (or at least it wasn't nbconverted after execution)

Reply:
I'll rerun this code to show the outputs, and then recommit

Reply:
Just pushed the reran code